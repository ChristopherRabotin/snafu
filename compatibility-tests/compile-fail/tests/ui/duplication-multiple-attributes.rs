use snafu::prelude::*;

#[derive(Debug, Snafu)]
enum EnumError {
    // Duplication on one line
    #[snafu(visibility(pub), visibility(pub))]
    AVariant,
}

fn main() {}
