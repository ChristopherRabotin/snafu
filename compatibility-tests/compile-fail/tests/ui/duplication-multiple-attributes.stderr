error: Multiple `visibility` attributes are not supported on an enum variant
 --> tests/ui/duplication-multiple-attributes.rs:6:30
  |
6 |     #[snafu(visibility(pub), visibility(pub))]
  |                              ^^^^^^^^^^^^^^^
